package.json

```diff
@@ -48,13 +48,14 @@
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.5",
     "eslint": "5.16.0",
-    "eslint-plugin-babel": "5.3.0",
     "eslint-config-wolox": "3.0.2",
     "eslint-config-wolox-react": "2.0.0",
-    "eslint-plugin-import": "2.17.3",
+    "eslint-plugin-babel": "5.3.0",
+    "eslint-plugin-import": "2.19.1",
     "eslint-plugin-jsx-a11y": "6.2.1",
     "eslint-plugin-prettier": "3.1.0",
     "eslint-plugin-react": "7.13.0",
+    "eslint-plugin-react-hooks": "^2.3.0",
```

Comment:
I don0t think we need this here, we need it in the main project but the bootstrap itself doesn't use react